packages/ui/components/InputCombobox/index.tsx

```
@@ -0,0 +1,328 @@
+    if (Array.isArray(localValue)) {
+      return localValue.map((item, idx) => {
+        const option = validOptions?.find((opt) => opt.value === item);
+
+        if (!option) {
+          return null;
+        }
+
+        return (
+          <>
+            {idx !== 0 && <span>,</span>}
+            <div className="flex items-center gap-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)